diff --git a/CMakeLists.txt b/CMakeLists.txt
index bfb5fe3..a308936 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -24,8 +24,8 @@ target_include_directories(VulkanMemoryAllocator-Hpp

 target_link_libraries(VulkanMemoryAllocator-Hpp
     INTERFACE
-        Vulkan::Vulkan
-        VulkanMemoryAllocator::VulkanMemoryAllocator
+        Vulkan::Headers
+        VulkanMemoryAllocator
 )

 install(
@@ -47,7 +47,7 @@ install(
 )

 install(
-    TARGETS VulkanMemoryAllocator-Hpp
+    TARGETS VulkanMemoryAllocator-Hpp VulkanMemoryAllocator
     EXPORT VulkanMemoryAllocator-HppTargets
 )
